OWNERS_ALIASES

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  core-ack-team:
+    - a-hilaly
+    - RedbackThomson
+  # TODO: Add your team members' GitHub aliases to the team alias
+  service-team: []
```

Comment:
This may require some changes.

Reply:
+1

Reply:
Can we remove RedbackThomson for now? i need to run a script to update them all over the org

Reply:
Sounds good. I will leave just you in that list for now.